### PR TITLE
Two small fixes from the demo project

### PR DESCRIPTION
**Pull request by doughsay**

### CODE REVIEW

File: config/dev.exs [hunk 1]
```diff
@@ -60,7 +60,7 @@ config :live_view_demo, LiveViewDemoWeb.Endpoint,
     patterns: [
       ~r"priv/static/.*(js|css|png|jpeg|jpg|gif|svg)$",
       ~r"priv/gettext/.*(po)$",
-      ~r"lib/live_view_demo_web/{live,views}/.*(ex)$",
+      ~r"lib/live_view_demo_web/(live|views)/.*(ex)$",
```
Comment (doughsay) [inline config/dev.exs:63]: This one was filed as a PR to the upstream demo project: https://github.com/DockYard/live_view_demo/pull/1

File: lib/live_view_demo_web/live/clock_live.ex [hunk 2]
```diff
@@ -20,6 +22,11 @@ defmodule LiveViewDemoWeb.ClockLive do
     {:noreply, put_date(socket)}
   end
 
+  def handle_event("boom", "", socket) do
```
Comment (doughsay) [inline lib/live_view_demo_web/live/clock_live.ex:25]: Clicking the clock sends this event, but there was no handler for it.

**Review by dhedlund — APPROVED** [review]

File: lib/live_view_demo_web/live/clock_live.ex (hunk 2 above)
Comment (kevinkoltz) [reply lib/live_view_demo_web/live/clock_live.ex:25]: I remember seeing this in the original demo where it shows how the front-end gracefully recovers after a crash
https://github.com/chrismccord/phoenix_live_view_example/blob/0b8e8ca080a142c5befa0f9c3c0e49956892df32/lib/demo_web/live/counter_live.ex#L8

Comment (doughsay) [reply lib/live_view_demo_web/live/clock_live.ex:25]: @kevinkoltz oooohhhh thanks!  That make more sense now.

**Pull request merged**